test/distributed/test_store.py

```diff
@@ -246,51 +241,28 @@ def _test_numkeys_delkeys(self, fs):
     def test_numkeys_delkeys(self):
         self._test_numkeys_delkeys(self._create_store())
 
-    def _create_client(self, index, addr, port, world_size, messages):
-        try:
-            client_store = dist.TCPStore(addr, port, world_size, timeout=timedelta(seconds=10))
-            self.assertEqual("value".encode(), client_store.get("key"))
-            client_store.set(f"new_key{index}", f"new_value{index}")
-            self.assertEqual(f"next_value{index}".encode(),
-                             client_store.compare_set(f"new_key{index}", f"new_value{index}", f"next_value{index}"))
-        except Exception:
-            messages.put('Caught exception: \n{}exiting process with exit code: {}'
-                         .format(traceback.format_exc(), MultiProcessTestCase.TEST_ERROR_EXIT_CODE))
-            sys.exit(MultiProcessTestCase.TEST_ERROR_EXIT_CODE)
+    def _create_client(self, index, addr, port, world_size):
+        client_store = dist.TCPStore(addr, port, world_size, timeout=timedelta(seconds=10))
+        self.assertEqual("value".encode(), client_store.get("key"))
+        client_store.set(f"new_key{index}", f"new_value{index}")
+        self.assertEqual(f"next_value{index}".encode(),
+                         client_store.compare_set(f"new_key{index}", f"new_value{index}", f"next_value{index}"))
 
     def _multi_worker_helper(self, world_size):
         addr = DEFAULT_HOSTNAME
         server_store = create_tcp_store(addr, world_size, wait_for_workers=False)
         server_store.set("key", "value")
         port = server_store.port
-        messages = mp.Queue()
-        processes = []
-        num_proccesses = random.randint(3, 5) if world_size == -1 else world_size
-        for i in range(num_proccesses):
-            p = mp.Process(target=self._create_client, args=(i, addr, port, world_size, messages))
-            processes.append(p)
-            p.start()
-        for p in processes:
-            p.join()
-        error_message = ""
-        while not messages.empty():
-            error_message += messages.get() + "\n"
-        if any([p.exitcode != 0 for p in processes]):
-            raise RuntimeError(error_message)
-
-    @unittest.skipIf(
-        IS_WINDOWS, "Skip test for windows due to multiprocessing library error when using windows spawn"
-    )
+        world_size = random.randint(5, 10) if world_size else world_size
```

Comment:
Not sure if this ternary operator is correct. Isn't it supposed to be `if world_size == -1`?

Reply:
Good catch, accidentally changed that. Fixed it now, thanks!